plugin/intero.vim

```diff
@@ -15,6 +15,10 @@ if !exists('g:intero_started')
     let g:intero_started = 0
 endif
 
+if !exists('g:intero_prompt_regex')
+    let g:intero_prompt_regex = '[^-]> '
+endif
```

Comment:
(I moved this to be initialized up front, rather than only once we get GHCi output, because we need it to define some match groups.)